sending/src/main/res/layout/balnce_action_bar.xml

```diff
@@ -9,8 +9,8 @@
 
     <ImageView
         android:id="@+id/closeX"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
+        android:layout_width="40dp"
```

Comment:
Try to import the vector and then set the size in dp before you will press next

Reply:
im idont want to change the code to wrap conetent with vectors assets